server/app/user.go

```diff
@@ -3,5 +3,14 @@ package app
 import "github.com/mattermost/focalboard/server/model"
 
 func (a *App) GetWorkspaceUsers(workspaceID string) ([]*model.User, error) {
-	return a.store.GetUsersByWorkspace(workspaceID)
+	users, err := a.store.GetUsersByWorkspace(workspaceID)
+	if err != nil {
+		return nil, err
+	}
+
+	for i := range users {
+		users[i].Email = ""
+	}
```

Comment:
Not a huge fan of this. I would say we can make a new store method that gets users without the email and call it here. Make the users model so that email is omitempty. This can become unnecessarily expensive if the number of users is huge ( O(n) ). Better to just retrieve users in store layer without email.

Reply:
Makes sense.